.github/workflows/codespell.yml

```diff
@@ -0,0 +1,21 @@
+name: codespell
```

Comment:
i am not sure why github workflow is started, is there any specific repo setting that disables the workflows?